GETTING_STARTED.md

```diff
@@ -19,6 +19,11 @@ Create a `.env` file, you can use the example one running:
 cp .env.example .env
 ```
 
+**NOTE**: When running the app in docker, different values are required for some
+environment variables (as per comments in .env.example). So, if running the app
```

Comment:
Are we ever running the app or tests without Docker?

Reply:
See comment above

Reply:
Not sure that answers my question I'm afraid - my question wasn't constrained by AWS. Do we ever run the admin backend application outside of Docker e.g., just by running `python app/main.py` in our terminal? Or is the development flow that we always use Docker for running our application?